geometry_msgs/Point position
geometry_msgs/Vector3 normal
---
uint32[] object_ids